packages/flutter/lib/src/widgets/dismissable.dart

```diff
@@ -340,17 +378,29 @@ class _DismissableState extends State<Dismissable> with TickerProviderStateMixin
       );
     }
 
-    Widget backgroundAndChild = new SlideTransition(
+    Widget content = new SlideTransition(
+      key: new Key('foreground'),
       position: _moveAnimation,
       child: config.child
     );
+
     if (background != null) {
-      backgroundAndChild = new Stack(
-        children: <Widget>[
-          new Positioned(left: 0.0, top: 0.0, bottom: 0.0, right: 0.0, child: background),
-          new Viewport(child: backgroundAndChild)
-        ]
-      );
+      List<Widget> children = <Widget>[];
+
+      if (!_moveAnimation.isDismissed) {
```

Comment:
Yeah, I think we're not super clear about when a widget is responsible for keeping sync between the widgets you hand in.  In this case, it makes sense that the background will be culled out when its not on screen, so it's sensible that it wouldn't always sync.  The loss of sync between the swipeaway and the collapse is probably less expected through...